server/src/main/resources/org/eclipse/lsp/cobol/service/delegates/completions/LanguageKeywords.txt

```diff
@@ -21,7 +21,7 @@ ABSTIME=
 ABSVAL=
 ACCELERATION=
 ACCELERATOR=
-ACCEPT= The ACCEPT statement transfers data or system date-related information into the data area referenced by the specified identifier.<br>[Read more](https://www.ibm.com/support/knowledgecenter/en/SS6SG3_6.2.0/com.ibm.cobol62.ent.doc/PGandLR/ref/rlpsacce.htm)<br>\u00A9 Copyright IBM Corporation 1994, 2019.<br>U.S. Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+ACCEPT= The ACCEPT statement transfers data or system date-related information into the data area referenced by the specified identifier.<br>[Read more](https://www.ibm.com/support/knowledgecenter/en/SS6SG3_6.2.0/com.ibm.cobol62.ent.doc/PGandLR/ref/rlpsacce.htm)<br>\u00A9 Copyright IBM Corporation 1994, 2019.<br>U.S. Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.<br>***<br>[IDMS] The ACCEPT statement transfers database, transaction, or task-related information to a specified location in program variable storage.
```

Comment:
Optional: I do not know how this URLs for docs are reliable (at least IBM COBOL docs are changing a lot).
Better to mention refer to IBM docs for details.

Reply:
We added these links because we've taken the descriptions from an internal source. I don't think we need to specify anything explicitly for the internal product.